MomotalkSource.theme.css

```diff
@@ -509,6 +511,10 @@
 				color: var(--brand-560);
 			}
 		}
+
+		&:has(article.embedWrapper_b558d0 .embedProvider_b0068a>a[href*="youtube"]) {
+			width: 97%;
+		}

```

Comment:
I'd check if `max-width: 432px;` really isn't needed. On my client it causes the embed containers to stretch to the full chat window width.

Btw your commit comments made me chuckle real good! :laughing:  No worries about borrowing my code it's not especially amazing css in my opinion and it slipped my mind I made those fixes :sweat_smile:

Reply:
😭 😭 I have failed.... But I'll go fix it really fast. Thank you again for your assistance.